src/main/java/lotto/LottoMain.java

```diff
@@ -19,31 +21,25 @@ public static void main(String[] args) {
         int ticketNumber = lottoPurchaseAmount / 1000;
 
         LottoGame lottoGame = new LottoGame();
-        Lottos lottos = lottoGame.purchase(ticketNumber);
+        LottoPurchaseResult lottoPurchaseResponse = lottoGame.purchase(LottoListGenerator.creatLottos(ticketNumber));
+        ResultView.printPurchaseLottos(lottoPurchaseResponse);
 
-        ResultView.printPurchaseLottos(lottos);
-        lottoGame.drawLotto(getWinnerLottoNumbers());
+        LottoDrawResult lottoResult = lottoGame.drawLotto(getWinnerLotto());
+        List<LottoWinnersDto> lottoWinnersDtos = lottoResult.createLottoWinnerDtos();
+        double yield = lottoResult.calculateYield(lottoPurchaseAmount);
 
-        List<LottoWinnersDto> lottoWinnersDtos = createLottoWinnerDtos(lottoGame);
         ResultView.printWinners(lottoWinnersDtos);
-        ResultView.printYield(lottoWinnersDtos, lottoPurchaseAmount);
+        ResultView.printYield(yield);
     }
 
-    private static List<LottoWinnersDto> createLottoWinnerDtos(LottoGame lottoGame) {
-        return Arrays.stream(Award.values())
-                .filter(award -> isBang(award))
-                .map(award -> new LottoWinnersDto(award, lottoGame.countWinners(award)))
-                .collect(Collectors.toList());
-    }
-
-    private static boolean isBang(Award award) {
-        return !award.equals(Award.BANG) && !award.equals(Award.UNIDENTIFIED);
-    }
-
-    private static Set<LottoNumber> getWinnerLottoNumbers() {
+    private static WinnerLotto getWinnerLotto() {
         Set<Integer> winnerNumbers = InputView.requestWinnerNumbers();
-        return winnerNumbers.stream()
+        Lotto lotto = new Lotto(winnerNumbers.stream()
                 .map(LottoNumber::new)
-                .collect(Collectors.toSet());
+                .collect(Collectors.toSet()));
+        Integer bonusNumber = InputView.requestBonusNumber();
+        LottoNumber bonusLottoNumber = new LottoNumber(bonusNumber);
```

Comment:
```suggestion
        int bonusNumber = InputView.requestBonusNumber();
        LottoNumber bonusLottoNumber = new LottoNumber(bonusNumber);
```
https://github.com/next-step/java-lotto/pull/1885#discussion_r684798586 에 대해서 다시 한번 고민해보시면 좋을 것 같습니다. 
`InputView.requestBonusNumber();`에서 리턴 타입도 마찬가지 입니다 :)

Reply:
넵 감사합니다!